flutter/build_fdroid.sh

```diff
@@ -207,14 +232,16 @@ prebuild)
 
 	cargo install \
 		cargo-ndk \
-		--version "${CARGO_NDK_VERSION}"
+		--version "${CARGO_NDK_VERSION}" \
+		--locked
 
 	# Install rust bridge generator
 
 	cargo install cargo-expand
```

Comment:
Lol, you forgot to add `--locked` to this line.

Reply:
You are right, will PR the change after I am back on laptop.